grpc/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>stargate</artifactId>
-    <version>1.0.23</version>
+    <version>1.0.24</version>
```

Comment:
```suggestion
    <version>1.0.25-SNAPSHOT</version>
```

Reply:
Thanks! I was coming here to point that out. :)